src/scenes/Services/components/ServicesItem/components/Block/Block.js

```diff
@@ -0,0 +1,70 @@
+import React, { useContext } from 'react';
+import { useStaticQuery, graphql } from 'gatsby';
+import PropTypes from 'prop-types';
+
+import { GlobalContext } from '@contexts';
+
+import styles from './Block.module.scss';
+
+const Block = ({ banner }) => {
+  const data = useStaticQuery(graphql`
+    query {
+      site {
+        siteMetadata {
+          email
+        }
+      }
+    }
+  `);
+  const { imagesAPI } = useContext(GlobalContext);
+  const images = imagesAPI.get([
+    'arrow-down.svg',
+    'services/desing/services-design-button-img.png',
```

Comment:
typo. `design` is a correct spelling.
add https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker to your vscode

Reply:
Thanks, this is what I was missing. This is basic feature in WebStorm.